independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanDeploymentValidator.java

```diff
@@ -23,11 +23,14 @@ default void validate(ValidationContext context) {
     }
 
     /**
+     * This method is deprecated and is not used internally.
      * 
      * @param target
      * @param rule
      * @return {@code true} if the given validation rule should be skipped for the specified target
+     * @deprecated This method will be removed at some point post Quarkus 1.4
      */
+    @Deprecated
```

Comment:
Do we have a substitute for this now or do we just drop it?

Reply:
I think that we should just drop it.

Reply:
Probably yes, I was just wondering since the deprecation doesn't mention it.